src/main/java/com/spotinst/sdkjava/model/ApiOceanGKEAutoScale.java

```diff
@@ -0,0 +1,54 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class ApiOceanGKEAutoScale {
```

Comment:
1) Follow same naming convention as given comment in the corresponding BL file.
2) Move this file to model.api.ocean.gke folder structure.

Reply:
Resolved